src/test/interface/generated_quantities_test.cpp

```diff
@@ -0,0 +1,119 @@
+#include <fstream>
+#include <gtest/gtest.h>
+#include <stdexcept>
+#include <string>
+#include <test/utility.hpp>
+
+using cmdstan::test::convert_model_path;
+using cmdstan::test::multiple_command_separator;
+using cmdstan::test::run_command;
+using cmdstan::test::run_command_output;
+
+class CmdStan : public testing::Test {
+public:
+  void SetUp() {
+    model_path.emplace_back("src");
```

Comment:
This is simpler in C++11 with

```
model_path = { "src", "test", "test-model", "gq_model" };
```

Same for other examples.

Reply:
done